css/index.css

```diff
@@ -101,10 +141,109 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
```

Comment:
On the boxes, the parent .boxes can do almost all the styling needed for these individual boxes.  You only want to override the color for each box like .box1, .box2.  Also, you can omit .middle-content .boxes from each of these since you have a class name for each box if you like.